tinker/templates/events/home.html

```diff
@@ -60,52 +60,57 @@
 
                 <div>
                     {# create an event button #}
-                    <a href="{{ url_for('events.EventsView:add') }}" id="event-create" class="btn btn-primary new-btn">
+                    <a href="add#page-title" id="event-create" class="btn btn-primary new-btn">
```

Comment:
the href seems weird?

Reply:
Yeah that is strange... I just changed it back to what it was before.